drivers/include/ina3221.h

```diff
@@ -225,6 +208,12 @@ typedef enum ina3221_enable_latch {
 } ina3221_enable_latch_t;
 #define INA3221_ENABLE_LATCH_MASK      (INA3221_ENABLE_WARN_LATCH  \
                                         | INA3221_ENABLE_CRIT_LATCH)  /**< Enable latch bit mask */
+
+/**
+ * @brief Flags @see ina3221_channel_t
+ */
+typedef int ina3221_ch_t;
```

Comment:
```suggestion
typedef unsigned ina3221_ch_t;
```

I personally try to avoid signed types for bit masks. E.g. the C standard allows e.g. `foo >> 4` on a signed foo to both sign and zero extend (arithmetic vs bitwise shift). While arithmetic shifts are nice for efficient division of negative numbers by powers of two, one really wants to avoid this for bitmasks.

(E.g. `((int8_t)0xff) >> 4` can have the value of `0xff` (arithmetic shift) and `0x0f`, while `((int8_t)0x0f) >> 3` has the result of `0x01` for both arithmetic and bitwise shift.)

Reply:
Ok